controllers/imageclusterinstall_controller.go

```diff
@@ -143,9 +138,8 @@ func (r *ImageClusterInstallReconciler) Reconcile(ctx context.Context, req ctrl.
 		return res, err
 	}
 
-	// Nothing to do if the installation process has already stopped
-	if installationCompleted(ici) {
-		log.Infof("Cluster %s/%s finished installation process, nothing to do", ici.Namespace, ici.Name)
+	// Nothing to do if the installation process started
+	if !ici.Status.BootTime.IsZero() {
 		return ctrl.Result{}, nil
```

Comment:
maybe move it before 
https://github.com/openshift/image-based-install-operator/pull/129/files#diff-2b950e24497af9d913e379e1aad24d9446d374d42e696b0ee2a0bd5079b3b5a2L131 ?

Reply:
hmm, you can't cause of finalizer but i believe there is no point to add reconcile log in case we are not actually?

Reply:
Lets add same logic to mapping functions? If we know we should not reconcile we can filter it earlier, at least for bmh and cd events, will make our queue smaller

Reply:
There's no mapping for the ICI itself.
Everything that run prior to this needs to happen regardless if the installation started or not.